OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -441,10 +441,10 @@ public void onLoadFinished(Loader<AsyncTaskResult<CourseComponent>> loader, Asyn
             loadData(validateCourseComponent(courseComponent));
             loadingIndicator.setVisibility(View.GONE);
             // Send a server call in background for refreshed data
-            getCourseComponentFromServer(false);
+            getCourseComponentFromServer(false, false);
         } else {
             // Course data is neither available in app session cache nor available in persistable cache
-            getCourseComponentFromServer(true);
+            getCourseComponentFromServer(true, false);
```

Comment:
here `forceRefresh` should be true cuz the app needs to get the data from the server.